ast.go

```diff
@@ -1098,16 +1150,21 @@ func (node *ColumnConstraintPrimaryKey) String() string {
 		constraintName = fmt.Sprintf("constraint %s ", node.Name.String())
 	}
 
-	if node.Order == ColumnConstraintPrimaryKeyOrderEmpty {
-		return fmt.Sprintf("%sprimary key", constraintName)
+	var autoIncrement string
```

Comment:
Adds the autoincrement logic to the `String` method so we execute the query with the autoincrement keyword enabled, even though the user did not use it.